src/V3Assert.cpp

```diff
@@ -505,6 +505,24 @@ class AssertVisitor final : public VNVisitor {
         } else if (nodep->displayType() == VDisplayType::DT_MONITOR) {
             nodep->displayType(VDisplayType::DT_DISPLAY);
             const auto fl = nodep->fileline();
+            AstNode* Monexp =  nodep->fmtp()->exprsp(); 
+            AstSenItem* MonSenItem = NULL;
+            int monsenflag =1;
+            while(Monexp != NULL)
+            {
+                if(Monexp->type() == VNType::atVarRef)
```

Comment:
```suggestion
                if (AstVarRef* varrefp = VN_CAST(monExprsp, NodeVarRef)) {
```

Then don't need cast below